compiler/frontend/spark/src/main/java/org/apache/nemo/compiler/frontend/spark/transform/GroupByKeyTransform.java

```diff
@@ -34,7 +34,7 @@
  * @param <K> key type.
  * @param <V> value type.
  */
-public final class GroupByKeyTransform<K, V> extends NoWatermarkEmitTransform<Tuple2<K, V>, Tuple2<K, Iterable<V>>> {
+public final class GroupByKeyTransform<K extends Serializable, V extends Serializable> extends NoWatermarkEmitTransform<Tuple2<K, V>, Tuple2<K, Iterable<V>>> {
```

Comment:
Unfortunately, this doesn't work when processing non-Serializable data elements.
Revert this change, and instead add a TODO to https://issues.apache.org/jira/browse/NEMO-431